src/EditorFeatures/CSharpTest/SignatureHelp/InvocationExpressionSignatureHelpProviderTests.cs

```diff
@@ -2485,5 +2485,46 @@ unsafe static void M()
 
             await TestAsync(markup, expectedOrderedItems);
         }
+
+        [Theory, CombinatorialData, Trait(Traits.Feature, Traits.Features.SignatureHelp)]
+        public async Task ShowWarningForOverloadUnavailableInRelatedDocument(bool typeParameterProvided)
+        {
+            var markup = $@"<Workspace>
+    <Project Language=""C#"" CommonReferences=""true"" AssemblyName=""Proj1"" PreprocessorSymbols=""TFM"">
+        <Document FilePath=""SourceDocument""><![CDATA[
+class C
+{{
+    public void M(object o) => false;
+#if TFM
```

Comment:
Okay, I have to ask what is probably a super basic question here, but what is "TFM" out of curiosity?

Reply:
TFM = target framework moniker